src/components/reviews-form/reviews-form.tsx

```diff
@@ -0,0 +1,35 @@
+// import { useState } from 'react';
+import FormRating from '../form-rating/form-rating';
+
+function ReviewsForm() {
+  // const [formData, setFormData] = useState({
+  //   textarea: ''
+  // });
+
+  // const handleFieldChange = (evt) => {
+  //   const { textarea, value } = evt.target;
```

Comment:
Надо будет поговорить об этом на звонке
```suggestion
  const { name, value } = evt.target;
```